### PR TITLE
Json reader test

### PR DESCRIPTION
**Pull request by Daniel-ND**

### CODE REVIEW

File: app/src/androidTest/java/com/google/droidjump/leveldata/JSONReaderTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class JSONReaderTest {
+    @Rule
+    public ActivityTestRule<MainActivity> activityTestRule = new ActivityTestRule<>(MainActivity.class);
+
+    @Test
+    public void getJSONObjectFromResource() throws JSONException {
```
Comment (saimonsaret) [inline app/src/androidTest/java/com/google/droidjump/leveldata/JSONReaderTest.java:43]: A good test for a general case. Can you try testing the edge cases as well? For example, when timeline array is empty. Also, what about finite and infinite levels?

**Review by saimonsaret — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/com/google/droidjump/leveldata/JSONReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        Resources resources = activityTestRule.getActivity().getResources();
+        int testfileId = R.raw.test;
+
```
Comment (redls) [inline app/src/androidTest/java/com/google/droidjump/leveldata/JSONReaderTest.java:46]: remove newline.

File: app/src/androidTest/java/com/google/droidjump/leveldata/JSONReaderTest.java (hunk 1 above)
Comment (saimonsaret) [reply app/src/androidTest/java/com/google/droidjump/leveldata/JSONReaderTest.java:42]: It is OK in this case, you could leave it like that.

**Review by saimonsaret — APPROVED** [review]
